src/styles/index.scss

```diff
@@ -1,3 +1,67 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  box-sizing: border-box;
+  padding: $calendar-padding;
+  width: calc((7 * $day-size) + (6 * $day-gap) + (2 * $calendar-padding));
+  gap: $day-gap;
+}
+
+.calendar__day {
+  position: relative;
+  width: $day-size;
+  height: $day-size;
+  box-sizing: border-box;
+  border: $day-border;
+  background-color: $day-color;
+
+  &::before {
+    content: '';
+    position: absolute;
+    transform: translate(-50%, -50%);
+    top: 50%;
+    left: 50%;
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+
+  &:hover {
+    cursor: pointer;
+    background-color: $day-hover-color;
+    transform: translate(0, -20px);
+    transition:
+      background-color 0.5s,
+      transform 0.5s;
```

Comment:
The `transition` property should be declared before the `&:hover` pseudo-class to ensure that the transition effect is applied to both hover on and hover off actions. This is a common best practice for CSS transitions.